src/Integration/Resources/Strings.resx

```diff
@@ -447,34 +447,6 @@ This is the general format to use when writing errors to output window or when t
 {0} - service
 {1} - http error code</comment>
   </data>
-  <data name="SonarLintProfileCheck" xml:space="preserve">
```

Comment:
Miscellaneous: deleted redundant strings that were used by the old QP comparison code, but that weren't cleaned up when that code was deleted.